instance/static/html/instance/index.html

```diff
@@ -50,7 +50,13 @@
                 |
                 <span>Studio: <a href="{{ selected.instance.studio_url }}" target="_new">{{ selected.instance.studio_url }}</a></span>
               </li>
-              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+              <li>
+                <a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> {{ selected.instance.branch_name }}
+                {% if selected.instance.github_pr_url %}
+                | <a href="{{ selected.instance.github_pr_url }}" target="_new">PR#{{ selected.instance.github_pr_number }}</a>
+                {% endif %}
+              </li>
+              {% endif %}
```

Comment:
@antoviaque The problems were mostly with myself, not with the instructions in the README.  The README could be a bit more detailed on what to put in the `.env` file and which settings actually need useful values to get the development server running.  (I simply copied `.env.test`, removed the `-test` suffix from the database name and ran `createdb opencraft`, which is probably the fastest way of getting a somewhat working setup.)